Library/Homebrew/formula.rb

```diff
@@ -1364,6 +1364,18 @@ def self.clear_installed_formulae_cache
     @installed = nil
   end
 
+  def self.comparison
```

Comment:
Sounds good. It's now in utils/comparison.rb, but extends `Utils` instead of creating a new module.